crates/bevy_math/src/bounding/bounded3d/primitive_impls.rs

```diff
@@ -323,29 +323,15 @@ impl Bounded3d for Triangle3d {
     /// The [`Triangle3d`] implements the minimal bounding sphere calculation. For acute triangles, the circumcenter is used as
     /// the center of the sphere. For the others, the bounding sphere is the minimal sphere
     /// that contains the largest side of the triangle.
-    fn bounding_sphere(&self, translation: Vec3, rotation: Quat) -> BoundingSphere {
-        if self.is_degenerate() {
+    fn bounding_sphere(&self, translation: Vec3, _: Quat) -> BoundingSphere {
```

Comment:
Not an expert on this, but maybe the unused parameter should be 
```suggestion
    fn bounding_sphere(&self, translation: Vec3, _rotation: Quat) -> BoundingSphere {
```
so it's better documented.